### DISCUSSION TITLE
unable to install oh my zsh using sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"

### DISCUSSION THREAD

**Discussion by Vishesh-Mistry** [ohmyzsh/ohmyzsh#11367] [Help] [answered]
I tried installing oh my zsh using the command
`sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
but its giving the error
_curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75218 ms: Operation timed out_
I've tried installing it multiple times but I keep getting the same error.

I also tried using
`sh -c "$(wget https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh -O -)"`
and I get the error
_Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2405:200:1607:2820:41::36|:443... failed: Operation timed out.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|49.44.79.236|:443... failed: Operation timed out._

I wasn't able to find the solution anywhere.

thanks in advance.

**Comment by carlosala** [comment, marked as answer]
It looks like an issue from your connection with github server, it has no relation with omz itself.

**Reply by Vishesh-Mistry** [reply]
okay, do you have any idea how can I fix the issue?

**Reply by carlosala** [reply]
Look for stack overflow and these kind of places, I don't know what exactly is.

**Reply by Vishesh-Mistry** [reply]
Thanks
I'm able to install oh my zsh using
`sh -c "$(curl -fsSL https://gitee.com/mcornella/ohmyzsh/raw/master/tools/install.sh)"`
